lib/callmonitor.js

```diff
@@ -211,7 +211,7 @@ function CallMonitor(adapter, devices, phonebook) {
         });
 
         client.connect({
-            host: adapter.config.ip,
+            host: adapter.config.ip || adapter.config.iporhost,
```

Comment:
YOu need to swap the order ... the new first, old as second (because when new is set old is no longer used)

Reply:
It is not needed to change the order, because the old setting will be resetted -> see index.html:309
change is comming soon

Reply:
see my other comment =)